### ISSUE TITLE
NTR: IL-1 alpha or beta high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8341]
New term request IL-1 alpha or beta high in blood (elthat_130516090131): IL1; interleukin 1 alpha or beta

**Comment by pnrobinson** [comment]
@MickeySegal
I think these should be two terms, or are these parameters typically measured together?

**Comment by MickeySegal** [comment]
This area seems to have changed dramatically over the past decade.  I no longer see any references to IL-1 alpha, so I changed our finding to only refer to IL-1 beta.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000374
- label: Increased circulating interleukin 1beta concentration
- parents: [Abnormal circulating interleukin concentration (HP:0030782)](https://hpo.jax.org/app/browse/term/HP:0030782)
- pmid: 
- def: An increased concentration of interleukin-1beta in the blood circulation.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8343
- synonyms:
